<?xml version="1.0"?>
<launch>
  <arg name="bag_file" default="/media/nikhilesh_ssd/Nikhilesh/demo_hackathon2024/moma_ui_2024-10-14-12-12-56.bag"/>
  <!-- <node name="rosbag_play" pkg="rosbag" type="play" args="$(arg bag_file) -l -/-topics /clock /tf /tf_static /rs_435_2/color/image_raw /rs_435_3/color/image_raw /elevation_mapping/elevation_map" output="screen"/> -->

  <!-- <include file="$(find moma_bringup)/launch/panda_real.launch"/> -->
  <!-- <include file="$(find moma_bringup)/launch/scan_box/scan_box_preproc.launch"/> -->
  <!-- <include file="$(find moma_bringup)/launch/scan_box/scan_box_sensors.launch"/> -->
  <!-- <include file="$(find moma_bringup)/launch/scan_box/scan_box_transformations.launch"/> -->
  <!-- <include file="$(find moma_bringup)/launch/scan_box/scan_box_preproc_new_w_voxblox.launch"/> -->
  
  <node name="ros_sam" pkg="ros_sam" type="sam_node.py" output="screen" args="--alsologtostderr" clear_params="true"/>
  <node name="moma_ui" pkg="moma_ui" type="moma_ui_node.py" output="screen" args="--alsologtostderr" clear_params="true"/>

</launch>